clientwrapper/atlasoperator/atlas_operator.go

```diff
@@ -432,6 +482,10 @@ func handleRequests(commandDelegatorApi ingest.CommandDelegatorApi, eventGenerat
 				var request requestdatatypes.ClientSelectiveSyncRequest
 				request = command.(requestdatatypes.ClientSelectiveSyncRequest)
 				err = selectiveSyncArgoApp(&request)
+			} else if command.GetEvent() == requestdatatypes.ClientMarkNoDeployRequestType {
+				var request requestdatatypes.ClientMarkNoDeployRequest
+				request = command.(requestdatatypes.ClientMarkNoDeployRequest)
+				handleNotificationRequest(request.RequestId, markNoDeploy, request)
```

Comment:
Should not pass through just act and pass through. Should be a final action that exists the flow.